squire_web/src/tournament/overview/mod.rs

```diff
@@ -27,16 +27,32 @@ impl Component for TournOverview {
             .unwrap()
             .state
             .query_tournament(&self.id, |t| {
+                let activeRounds = t.round_reg.rounds.values().filter(|r| r.is_active()).count();
+                let certifiedRounds = t.round_reg.rounds.values().filter(|r| r.is_certified()).count();
+                let registeredPlayers = t.player_reg.players.values().filter(|p| p.can_play()).count();
+                let droppedPlayers = t.player_reg.players.values().filter(|p| !p.can_play()).count();
```

Comment:
Again, you can get away with a single pass through the players in the same way they you can with rounds or by calculating `droppedPlayers` from the total player count
```suggestion
                let dropped = t.player_reg.len() - active;
```